kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -212,4 +309,33 @@
   .sign-in-error
     color: red
 
+  .suggestions
+    background-color: white
+    box-shadow: 1px 2px 8px darken(white, 10%)
+    color: $core-text-default
+    display: block
+    list-style-type: none
+    margin: 0
+    width: 100%
+    padding: 0
+    z-index: 10
```

Comment:
Should be 8. https://material.io/guidelines/material-design/elevation-shadows.html#elevation-shadows-elevation-android